### ISSUE TITLE
Adding rotation spoofers

### ISSUE THREAD

**Issue by bad-lang** [PrismarineJS/mineflayer#1995]
## Is your feature request related to a problem? Please describe.
i really hate the "[killaura](https://www.youtube.com/watch?v=walbRk20KYU)" (looking at a mob and swinging weapon) currently implemented. For anarchy servers like 2b, killaura is a must-have and for mineflayer bots it's no different. 

## Describe the solution you'd like
A rotation faker using fake packets. Examples:

https://github.com/Wurst-Imperium/Wurst7/blob/master/src/main/java/net/wurstclient/RotationFaker.java
https://github.com/bebeli555/CookieClient/blob/main/src/main/java/me/bebeli555/cookieclient/utils/RotationUtil.java

## Describe alternatives you've considered
N/A

## Additional context
N/A

**Comment by DrMoraschi** [comment]
You could just send an attack packet

**Comment by bad-lang** [comment]
@DrMoraschi could you please provide an example? thanks :)

**Comment by aesthetic0001** [comment]
won't bot.attack suffice

**Comment by DrMoraschi** [comment]
> @DrMoraschi could you please provide an example? thanks :)

You write a packet with bot._client, look at how bot.attack is implemented in the source code

**Comment by Zn10plays** [comment]
This issue is not with minelayer but with your current implementation.

**Comment by rom1504** [comment]
please explain better what you want if you care about this and reopen
